Juego de Piedra Papel o Tijera

marcof2une opened the pull request.

marcof2une committed:
Creacion de Carpetas

marcof2une committed:
Merge branch 'mmasias:main' into main

marcof2une committed:
Commit borrando archivos y moviendo carpetas

marcof2une committed:
Moviendo carpetas

marcof2une committed:
Crer carpeta Destino

marcof2une committed:
Borrando place holders de creacion de carpetas

marcof2une committed:
Creacion de archivos de texto en origen

marcof2une committed:
Texto de los archivos modificado

marcof2une committed:
Los archivos han sido movidos de CarpetaOrigen a CarpetaDestino

marcof2une committed:
Archivos modificados nuevamente ya en carpetaDestino

marcof2une committed:
Merge branch 'mmasias:main' into main

marcof2une committed:
Cambios solictados por manuel masias realizados.

marcof2une committed:
Q viva el hola mundo

marcof2une committed:
Merge branch 'mmasias:main' into main

marcof2une committed:
Merge branch 'main' of https://github.com/marcof2une/MPRG1

marcof2une committed:
Merge pull request #1 from marcof2une/main

Actualizando cambios de main en MMASIAS a develop

marcof2une committed:
Entrega de archivo de nota final. Repo equivocado

marcof2une committed:
Merge branch 'mmasias:main' into develop

marcof2une committed:
Moviendo archivos a carpeta adecuada.

marcof2une committed:
Mini prueba de adivina numero

mmasias committed:
Dos ejemplos...

mmasias committed:
Tres ejemplos ternarios

mmasias committed:
Enunciado de los ejemplos

mmasias committed:
Enlace en la página de ejercicios

mmasias committed:
Incluye salida esperada en contenido de ejercicios

mmasias committed:
Update ClasificacionEdad.md

mmasias committed:
Definición de reto 001

mmasias committed:
Refinamos requisito (Human vs Computer)

mmasias committed:
Una imagen...

marcof2une committed:
Primera version de piedra pel tijera

marcof2une committed:
JuegoTerminado

marcof2une committed:
Merge branch 'mmasias:main' into main

mmasias commented on entregas/figueroaMarco/src/PiedraPapelTijera.java:
```diff
@@ -0,0 +1,30 @@
+        System.out.println("Como jugar? Ingresa T para Tijera, P para Piedra y Pa para Papel");
+        Scanner ScannerInput = new Scanner(System.in);
+        
+        final String Tijera = "T";
+        final String Piedra = "P";
+        final String Papel = "Pa";
```
```suggestion
        final String TIJERA = "Tijera";
        final String PIEDRA = "Piedra";
        final String PAPEL = "Papel";
```

mmasias commented on entregas/figueroaMarco/src/PiedraPapelTijera.java:
```diff
@@ -0,0 +1,30 @@
+        String resultado;
+        
+        numeroOrdernador = (int) (Math.random() * (3) + 1);
+        comparador = numeroOrdernador == 1 ? Tijera : numeroOrdernador == 2 ?  Piedra : numeroOrdernador == 3 
+        ? Papel : "Numero no establecido";
+        inputUsuario = ScannerInput.nextString();
+        resultado = comparador == inputUsuario ? "Empataron" : (inputUsuario == "T" && comparador == "Pa") ? 
+        "Ganaste" : (inputUsuario == "T" && comparador == "P") ? "Perdiste" : 
+        (inputUsuario == "P" && comparador == "T") ? "Ganaste" : (inputUsuario == "P" && comparador == "Pa")
+         ? "Perdiste" : (inputUsuario == "Pa" && comparador == "T") ? "Perdiste" : (inputUsuario == "Pa" && comparador == "P")
+         ? "Ganaste" : "Secuencia no establecida";
+         
```
Indente y utilice espacios para mejorar legibilidad. Eso es muy dificil de seguir

marcof2une committed:
Mini test de for loop

marcof2une committed:
Entrega caracol

mmasias committed:
Código!

mmasias committed:
Caracol ternario!

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Recopilación y ordenado de ejercicios

mmasias committed:
Create .gitignore

mmasias committed:
Ejercicios++

mmasias committed:
Update README.md

mmasias committed:
🪨📄✂️🦎🖖🏻

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Un dia en la vida de Marco
Pendiente:
- La madre
- Los dias

marcof2une committed:
Commit Asistencia

mmasias committed:
Solucion hecha en clase

mmasias committed:
Update ViajeMarco.java

Es más lógico expresarlo así:

seHanEncontrado si la distancia entre Marco y su madre es menor o igual a cero.

mmasias committed:
Propuesta de solución fullTernario 💪🏼💪🏼💪🏼

mmasias committed:
Actualización con soluciones

mmasias committed:
CarreraCamellos hecho en clase

mmasias committed:
Publicada la solución propuesta de CarreraCamellos

mmasias committed:
Extensión

mmasias committed:
Avance de clase. Inicio de solución de la Batalla

mmasias committed:
Update README.md

mmasias committed:
Versión minimalista
Vídeo de construcción en https://www.youtube.com/watch?v=bebuAZLYAVE

mmasias committed:
Update README.md

marcof2une committed:
repaso no commit en main

mmasias committed:
Ejercicio del centro comercial
Además incluyo una corrección en el ejercicio de la carrera de camellos.

mmasias committed:
Solución (hecha en clase) del centro comercial

mmasias committed:
Fix & version ASCII

mmasias committed:
Update README.md

mmasias committed:
Fix, inclusión de hora/minuto y animación

marcof2une committed:
Merge branch 'mmasias:main' into main

marcof2une committed:
Primer commit con capacidad de pasar a la gente de la caja a la fila y procesamiento de productos

marcof2une committed:
CarreraCamellos

marcof2une committed:
Merge branch 'main' of https://github.com/marcof2une/MPRG1

mmasias committed:
Segundo intento

mmasias committed:
3rd

mmasias committed:
Enlaces

marcof2une committed:
Commit del ejercicio del parcial

marcof2une committed:
Un clase1112 basico

marcof2une committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Centro comercial resuelto con métodos estáticos
No he puesto commits individuales :( Disculparme

marcof2une committed:
Merge branch 'mmasias:main' into develop

marcof2une committed:
Merge pull request #2 from marcof2une/main

Añadir cambios de develop a main

marcof2une committed:
Declaracion inicial y creando metodos.

marcof2une committed:
Agregadas todas las funcionalidades.

marcof2une committed:
Merge pull request #3 from marcof2une/develop

Pull Request to main

mmasias committed:
eP2

marcof2une committed:
Merge branch 'mmasias:main' into main

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Ejercicio de temperatura de 7 dias
- Con métodos estáticos
- Adecuado reparto de responsabilidades

mmasias committed:
Refactoriza manejo de temperaturas a matrices

- Cambia variables individuales de temperatura por una matriz.
- Actualiza métodos calcularMinima, calcularMaxima y calcularMedia para usar la nueva matriz.
- Simplifica el código de solicitud de temperaturas con un bucle for.

mmasias committed:
Pequeñas anécdotas acerca de una matriz

mmasias committed:
Cambio extra

marcof2une committed:
Update of file

marcof2une committed:
Created file with matrix

Added a test with a simple ascii printing

marcof2une committed:
Changed a few characters

JavierLianoRioz committed:
Update README.md

mmasias committed:
Merge pull request #689 from JavierLianoRioz/patch-1

Modificación del Readme

mmasias committed:
pyMundo y su solución

mmasias committed:
Update aDiaDeHoy.md

mmasias committed:
Update README.md

mmasias committed:
Update README.md

marcof2une committed:
Codigo mugre a ciorregir del reto de la herradura.

https://github.com/puntoReflex/.github/blob/main/retos%26proyectos/juegoHerradura.md

Commit:
Juego de la herradura terminado.

mmasias committed:
Ja!

marcof2une committed:
Merge pull request #5 from mmasias/patch-1

Ja! Añadido cambios de manuel

marcof2une committed:
Estudio de IGPS

marcof2une committed:
IGPS.md

marcof2une committed:
Update IGPS.md

Commit:
Retos en casa

marcof2une committed:
Primeros 3 ejercicios de ForIf

marcof2une committed:
Siguiente avance del reto for if.

marcof2une committed:
Avance Reto for if, 8 no funciona.

marcof2une committed:
Mas avances en el reto for if.

marcof2une committed:
Archivo de duda para reto ClasificacionConductor

marcof2une committed:
Reto Clasificacion Conductor con metodos. Solo hecho parcialmente.

marcof2une committed:
Iteracion de clasificado de conductor.

Posterior a la respuesta de la consulta en el repo, decidi hacer otra iteracion siguiendo una linea similar a la ya comenzada pero utilizando el poder de las referencias de arrays como parametros de 1 funcion principal.

marcof2une committed:
Terminada la funcion de cllasificacion de edades.

marcof2une committed:
Funcion que imprime el output.

marcof2une committed:
Correcion final para presentacion.

marcof2une committed:
Merge branch 'mmasias:main' into main

marcof2une committed:
Una iteracion mas al programa de clasificacion de conductor.

Pendiente resolver error de array out of bouns.

mmasias committed:
Update README.md

mmasias committed:
Update README.md

marcof2une committed:
Merge pull request #6 from marcof2une/develop

Updating changes done in develop to main

marcof2une committed:
Merge branch 'mmasias:main' into main

mmasias committed:
fixLink

marcof2une committed:
Merge branch 'mmasias:main' into main
